modules/distr-tracing-config-ingester.adoc

```diff
@@ -42,7 +42,7 @@ The deadlock interval is disabled by default (set to `0`), to avoid terminating
 |options:
   log-level:
 |Logging level for the Ingester.
-|Possible values: `trace`, `debug`, `info`, `warning`, `error`, `fatal`, `panic`.
+|Possible values: `debug`, `info`, `warning`, `error`, `fatal`, `panic`.
```

Comment:
@JStickler AFAIK, `warning` is not supported. It should be `warn` [1] and this is applicable for all the components, except `jaeger-operator`, AFAIK, `jaeger-operator` not using `zap` logger.

```
$ podman run -e SPAN_STORAGE_TYPE=cassandra jaegertracing/jaeger-ingester:1.31 --help
--log-level string                                Minimal allowed log Level. For more levels see https://github.com/uber-go/zap (default "info")
```

[1] - https://github.com/uber-go/zap/blob/v1.21.0/zapcore/level.go#L140~L153

@pavolloffay please confirm

Reply:
Updated